### PR TITLE
feat: selectable auto call outs

### PR DESCRIPTION
**Pull request by tracernz**
Fixes #[issue_no]

## Summary of Changes

Radio altitude auto call outs are now selectable, subject to Airbus rules:
- one of "five hundred" or "four hundred" is mandatory,
- only one of "two thousand five hundred" and "twenty five hundred" can be selected.

~~Draft while waiting for the twenty five hundred sound to be added.~~ Boris will replace the twenty five hundred sample with a new one 🔜 .

An LVar is added in this PR but not currently used. This is for @beheh's new FWC.

## Screenshots (if necessary)

image

## Additional context

Discord username (if different from GitHub):

## Testing instructions

Try adjusting the auto call out options under Settings -> Aircraft Options / Pin Programs -> Automatic Call Outs. Make sure they work as expected (subject to the bugs with the current auto callout system). Make sure the two rules listed above are enforced by the UI.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: fbw-a32nx/docs/a320-simvars.md [hunk 1]
```diff
@@ -1231,6 +1231,27 @@
     - Persistent
     - Enables developer-specific options like direct payload adjustments
 
+- A32NX_RADIO_AUTO_CALL_OUTS
```
Comment (Benjozork) [inline fbw-a32nx/docs/a320-simvars.md:1234]: **suggestion:** name this LVar to something more specific

File: fbw-a32nx/src/localization/flypad/en.json [hunk 2]
```diff
@@ -489,10 +489,31 @@
       "PaxSigns": "PAX Signs",
       "RmpVhfSpacing": "RMP VHF Spacing",
       "Satcom": "Satcom",
+      "Select": "Select",
       "ThrustReductionHeight": "Thrust Reduction Height (ft)",
       "Title": "Aircraft Options / Pin Programs",
       "WeightUnit": "Weight Unit"
     },
+    "AutomaticCallOuts": {
```
Comment (Benjozork) [inline fbw-a32nx/src/localization/flypad/en.json:497]: **thought:** I don't think we want the individual call-outs to be translated, as those are literally the vocalization you hear

**Review by Benjozork — APPROVED** [review]

**Comment by NathanInnes** [comment]
IMO I would prefer to see numbers instead of it being spelled out, would also save time on translating

**Comment by tracernz** [comment]
> IMO I would prefer to see numbers instead of it being spelled out, would also save time on translating

That's not going to work when there are multiple pronunciations e.g. "two thousand five hundred" vs. "twenty five hundred".

**Comment by auroraisluna** [comment]
> > IMO I would prefer to see numbers instead of it being spelled out, would also save time on translating
> 
> That's not going to work when there are multiple pronunciations e.g. "two thousand five hundred" vs. "twenty five hundred".

Perhaps a parenthesis that has the number? Or `number - text`  (so e.g. "2000 - Two Thousand") 
also, this could be changed in localization technically 😛

**Comment by auroraisluna** [comment]
QA Report

Discord: alepouna
Object of testing: #8167
Tier of Testing : 1
Date : 19/08/2023

Testing Process:
- Airbus Rules:
   - [PASS] Can only have either "Two Thousand Five Hundred" OR "Twenty Five Hundred" OR none. 
   - [PASS] Must have either "Five Hundred" or "Four Hundred". 
- Callouts heard: 
   - [PASS] 2,500 
   - [PASS] 20 500
   - [FAIL] 2000
   - [PASS] 1000
   - [PASS] 500
   - [PASS] 400
   - [PASS] 300
   - [PASS] 200
   - [PASS] 100
   - [PASS] 50
   - [PASS] 40
   - [PASS] 30
   - [PASS] 20
   - [PASS] 10
   - [PASS] 5

Testing Results:
Not Passed

Negatives:
2000 was not audible. 

Notes:
I have also tested all off, various configurations settings, etc, to confirm they are operable by the EFB. Other than the 2000, nice :D

**Comment by auroraisluna** [comment]
QA Report

Discord: alepouna
Object of testing: #8167
Tier of Testing : 1
Date : 19/08/2023

Testing Process:
- Airbus Rules:
   - [PASS] Can only have either "Two Thousand Five Hundred" OR "Twenty Five Hundred" OR none. 
   - [PASS] Must have either "Five Hundred" or "Four Hundred". 
- Callouts heard: 
   - [PASS] 2,500 
   - [PASS] 20 500
   - [PASS] 2000
   - [PASS] 1000
   - [PASS] 500
   - [PASS] 400
   - [PASS] 300
   - [PASS] 200
   - [PASS] 100
   - [PASS] 50
   - [PASS] 40
   - [PASS] 30
   - [PASS] 20
   - [PASS] 10
   - [PASS] 5

Testing Results:
Passed

Notes:
Dunno if this was because of terrain on LGAV (03LY ILS) but 2000 played twice in quick succession (with tops 2 second delay), could not replicate in other airports so I assume its normal

**Comment by frankkopp** [comment]
@tracernz pls remember to add an issue to docs if you add/change stuff on the EFB (and on other documented features) - or even better write the docs update and send a PR 😉 - I created an issue for this one already

**Pull request merged**